src/App.tsx

```diff
@@ -1,14 +1,73 @@
-/* eslint-disable max-len */
-import React from 'react';
+/* eslint-disable no-console */
+import React, { useEffect, useMemo, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-
+import { useDispatch } from 'react-redux';
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { getTodos } from './api';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { useAppSelector } from './app/hooks';
+import { actions as todosActions } from './features/todos';
+import { Select } from './types/Select';
+import { actions as currentTodoActions } from './features/currentTodo';
 
 export const App: React.FC = () => {
+  const dispatch = useDispatch();
+  const todos = useAppSelector(state => state.todos);
+  const selectedValue = useAppSelector(state => state.filter.status);
+  const inputValue = useAppSelector(state => state.filter.query);
```

Comment:
Please, create selectors and call them here

Reply:
I didnt catch it